gammapy/obs/observation.py

```diff
@@ -128,6 +141,8 @@ def select_range(self, selection_variable, value_range, inverted=False):
 
         # build and apply mask
         mask = (value_range[0] <= value) & (value < value_range[1])
+        if np.allclose(value_range[0], value_range[1]):
+            mask = value_range[0] == value
```

Comment:
I know it's equivalent, but I find adding extra parentheses makes the code a bit more readable:
```python
mask = (value_range[0] == value)
```